.github/workflows/test-ruby.yml

```diff
@@ -19,15 +19,8 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
-    strategy:
-      fail-fast: true
-      matrix:
-        mode:
-          - production
-          - test
     env:
-      RAILS_ENV: ${{ matrix.mode }}
-      BUNDLE_WITH: ${{ matrix.mode }}
+      RAILS_ENV: production
```

Comment:
Wait, shouldn't the `RAILS_ENV` be `test`?

Reply:
My bad 🤦 

I thought it would be the same for the JS build, but probably not because we want the env variables from `.env.test` to be used for this build… but then, it means we no longer test the real production build?

Reply:
I prefered to keep the 2 builds, as they may be slightly different.